package.json

```diff
@@ -64,9 +60,9 @@
   "dependencies": {
     "lodash.isequal": "^4.5.0",
     "prop-types": "^15.6.1",
-    "react-dnd": "3.0.2",
-    "react-dnd-html5-backend": "3.0.2",
-    "react-dnd-scrollzone": "^4.0.0",
+    "react-dnd": "^6.0.0",
+    "react-dnd-html5-backend": "^6.0.0",
+    "react-dnd-scrollzone": "github:dolezel/react-dnd-scrollzone.git#41203ef00c991f744aa87e897d7d518d32f6a994",
```

Comment:
Do you think in the future you can publish your own package? maybe a merge request to fix that library?

Reply:
I submitted PR, but do not expect it will be merged. 🤷‍♂️